src/domain/entities/DataSet.ts

```diff
@@ -40,6 +73,15 @@ export class DataSet extends Struct<DataSetAttrs>() {
         return DataSet.create({ ...this, orgUnits });
     }
 
+    static buildOrgUnitsFromPaths(paths: string[]): OrgUnit[] {
+        const orgUnits = paths.map(path => ({
+            id: _(path.split("/")).last() || "",
+            name: path,
+            path: path.split("/").slice(1),
```

Comment:
In our domain, a path is an array of IDs, right? But these seem DHIS2 paths, so this processing should not be in the domain layer.